Header header
uint16 time_domain_bins
uint16 num_chirps
float32[] micro_doppler_array